be/src/runtime/snapshot_loader.cpp

```diff
@@ -498,7 +498,8 @@ Status SnapshotLoader::primary_key_move(const std::string& snapshot_path, const
     }
     snapshot_meta.tablet_meta().set_tablet_id(tablet_id);
 
-    RETURN_IF_ERROR(SnapshotManager::instance()->assign_new_rowset_id(&snapshot_meta, snapshot_path));
+    RETURN_IF_ERROR(
+            SnapshotManager::instance()->assign_new_rowset_id(&snapshot_meta, snapshot_path, tablet->tablet_schema()));
```

Comment:
This function is only used for primary table backup/restore clone. Currently, inverted index does not support pk table. It seems that no need to change this function?